doc/nrf/releases_and_maturity/known_issues.rst

```diff
@@ -3326,6 +3326,15 @@ NCSDK-22799: Assert when requesting clock from the NFC interrupt context
 
   **Workaround:** To fix the issue, disable the :kconfig:option:`CONFIG_NFC_ZERO_LATENCY_IRQ` Kconfig option.
 
+nrf_security
```

Comment:
I have now added this known issue to my PR https://github.com/nrfconnect/sdk-nrf/pull/17997. Please remove from this PR.

Reply:
Sure, will do that!

Reply:
Ok don, please remove the nack @peknis